google/cloud/aiplatform/models.py

```diff
@@ -314,6 +316,266 @@ def deploy(
 
         return endpoint
 
+    def batch_predict(
+        self,
+        job_display_name: str,
+        gcs_source: Optional[Sequence[str]] = None,
+        bigquery_source: Optional[str] = None,
```

Comment:
Should these merged into a single parameter called `source` that distinguished between the two based on prefix? I.e. `bq://` or `gs://`